### PR TITLE
Print design 1

### PR DESCRIPTION
**Pull request by flob38**
Improvement of printed pages : 

- Reduce margin and font-size
- Remove some icon and replace with text when useful

### CODE REVIEW

**Comment by flob38** [comment]
I'll be off until monday, so I wouldn't be able to work on this before.

Image : I'll remove this part and discuss on the forum.

**Comment by cbeauchesne** [comment]
Ok. And btw, even if it's obvious, thank's a lot :)

**Comment by flob38** [comment]
There's something wrong : got a page-break after some document's title
http://localhost:8080/routes/50947/fr/dome-de-la-lauze-traversee-glacier-de-la-girose-vallon-de-la-selle => problem
http://localhost:8080/routes/53940/fr/mont-aiguille-la-tour-des-gemeaux => ok
Any idea?

File: src/components/generics/Activities.vue [hunk 1]
```diff
@@ -16,3 +16,22 @@
     }
   };
 </script>
+
+<style lang="scss">
```
Comment (cbeauchesne) [inline src/components/generics/Activities.vue]: Small, but important detail : `<style scoped lang="scss">` :)

**Pull request merged**